lib/oeqa/selftest/updater.py

```diff
@@ -509,4 +454,27 @@ def qemu_send_command(port, command):
     stdout, stderr = s2.communicate()
     return stdout, stderr, s2.returncode
 
+
+def akt_native_run(testInst, cmd, **kwargs):
```

Comment:
I think I'm a fan of writing utility functions like this. I hadn't looked into it too much yet but seems like it should make life easier.